opencti-platform/opencti-front/src/private/components/arsenal/Malwares.tsx

```diff
@@ -179,9 +183,11 @@ const Malwares = () => {
     <>
       <Breadcrumbs variant="list" elements={[{ label: t_i18n('Arsenal') }, { label: t_i18n('Malwares'), current: true }]} />
       {viewStorage.view !== 'lines' || !dataTableEnabled ? renderCards() : renderList()}
```

Comment:
Sorry, I really forgot to tell you guys that I've added a new "Table" in those views (Malware and all Threats)
Could you add the Create Button in the new views ? Should be pretty straight forward with the "additionalHeaderButtons" attribute.